src/sympc/store/crypto_primitive_provider.py

```diff
@@ -13,6 +15,9 @@ class CryptoPrimitiveProvider:
     """A trusted third party should use this class to generate crypto primitives."""
 
     _func_providers: Dict[str, Callable] = {}
+    _ops_list: DefaultDict[str, List] = DefaultDict(list)
```

Comment:
I had no idea we could use the DefaultDict from typing :D.

I thought we would need to do and use that one.
```from collections import defaultdict```

and use this one